serverless.yml

```diff
@@ -44,39 +44,43 @@ custom:
     - binance
     - gemini
     - okex
+    - kraken
```

Comment:
Sorry this is mostly just the linter being a pain. I did make a meaningful change here, lines 68-69 and line 77 though.